topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -130,78 +294,191 @@ In general, quality treatments include:
     - beginning/end of sequence
     - removing adapters
 
-To improve the overall sequence quality, we use the [Trim Galore!](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) tool. This tool enhances sequence quality by automating adapter trimming as well as quality control.
+To accomplish this task we use the [Trim Galore!](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) tool. This tool enhances sequence quality by automating adapter trimming as well as quality control.
 
 > ### {% icon hands_on %} Hands-on: Improvement of sequence quality
 >
-> 1. **Trim Galore!** {% icon tool %}: Run **Trim Galore! Quality and adapter trimmer of reads** on the imported FastQ file
+> 1. Run **Trim Galore!** {% icon tool %} with the following parameters
+>    - *"Is this library paired- or single-end?"*: `Single-end`
+>       - {% icon param-file %} *"Reads in FASTQ format"*: `reads_1` (Input dataset)
+>
+>          > ### {% icon tip %} Tip: Not selectable files?
+>          > If your FASTQ files cannot be selected, you might check whether their format is FASTQ with Sanger-scaled quality values (`fastqsanger`). You can edit the data type by clicking on the pencil symbol.
+>          {: .tip}
+>
+>       - *"Adapter sequence to be trimmed"*: `Automatic detection`
+>
+>          If you know which adapter sequence was used during library preparation, provide its sequence. Otherwise use the option for automatic detection and trimming of adapter sequences.
+>
+>    - *"Trim Galore! advanced settings"*: `Full parameter list`
+>       - *"Trim low-quality ends from reads in addition to adapter removal (Enter phred quality score threshold)"*: 20
+>         
+>          To time low-quality ends (below 20) from reads in addition to adapter removal
+>
+>       - *"Overlap with adapter sequence required to trim a sequence"*: `5`
+>   
+>          The default value "1" is too stringent, and on average 25% of reads will be trimmed. Please set it to 5 bases to loosen the required overlap
```

Comment:
Setting 5 bases for the overlap here makes this inconsistent with other tutorials e.g. the ref-based RNA-seq as the default Cutadapt overlap of 3 is used there. Should this also be 3? 
Separate minor point, the ref-based tutorial has replaced Trim Galore with Cutadapt do you think should that also be done here for consistency? The Cutadapt wrapper is more up to date I think than the Cutadapt version used in the current Trim Galore wrapper (although there are new bioconda versions of Trim Galore available).